new/src/sections/portfolio/portfolio.scss

```diff
@@ -0,0 +1,452 @@
+$background-image-url: "../../assets/background/bg.png";
+// 768 1024 1080 1536
+
+.portfolio {
+  background-image: url(#{$background-image-url}) !important;
+  background-size: 91.5%;
+  width: 100%;
+  height: 100%;
+  padding: 3rem 0;
+
+  @media (max-width: 640px) {
+    background-size: 100%;
+    padding: 2rem 0;
+  }
+}
+
+.portfolio .row {
+  --bs-gutter-x: 1.5rem;
+  --bs-gutter-y: 0;
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: calc(var(--bs-gutter-y) * -1);
+  margin-right: calc(var(--bs-gutter-x) * -0.5);
+  margin-left: calc(var(--bs-gutter-x) * -0.5);
+  justify-content: center;
+  gap: 2.08rem;
+
+  @media (max-width: 640px) {
+    --bs-gutter-x: 0.5rem;
+  }
+}
+
+.portfolio .row > * {
+  flex-shrink: 0;
+  width: 100%;
+  max-width: 100%;
+  padding-right: calc(var(--bs-gutter-x) * 0.5);
+  padding-left: calc(var(--bs-gutter-x) * 0.5);
+  margin-top: var(--bs-gutter-y);
+}
+
+.container,
+.container-fluid,
+.container-xxl,
+.container-xl,
+.container-lg,
+.container-md,
+.container-sm {
+  width: 100%;
+  padding-right: var(--bs-gutter-x, 0.75rem);
+  padding-left: var(--bs-gutter-x, 0.75rem);
+  margin-right: auto;
+  margin-left: auto;
+  height: fixed;
+}
+
+@media (min-width: 640px) {
+  .container-sm,
+  .container {
+    max-width: 540px;
+  }
+}
+
+@media (min-width: 768px) {
+  .container-md,
+  .container-sm,
+  .container {
+    max-width: 720px;
+  }
+}
+
+@media (min-width: 1024px) {
+  .container-lg,
+  .container-md,
+  .container-sm,
+  .container {
+    max-width: 1024px;
+  }
+}
+
+@media (min-width: 1280px) {
+  .container-xl,
+  .container-lg,
+  .container-md,
+  .container-sm,
+  .container {
+    max-width: 1280px;
+  }
+}
+
+@media (min-width: 1536px) {
+  .container-xxl,
+  .container-xl,
+  .container-lg,
+  .container-md,
+  .container-sm,
+  .container {
+    max-width: 1520px;
+  }
+}
+
+.portfolio .header-portfolio {
```

Comment:
Class name `header-portfolio` may be confused with `Header` component, whose classnames start with `header`